### PR TITLE
#5 Project: Smashing Magazine Teardown (HTML + CSS)

### PR DESCRIPTION
**Pull request by sergioobolevich**
I made the project with mixed text and div placeholders, all in white and black color palette.
The changes I have made as follows:

- First HTML + CSS Layout sketch
- Added Linters
- Finished Headers Layout
- Moved CSS and IMG to folders
- Set the article's layout
- Added improvements to articles and header layout
- Added remaining sections: Community links, book offer
- Fixed layout and CSS errors
- Finished footer section
- Improved design, classes, and HTML code again.
- Added newsletter form and styled it
- Made more fixes and changed fontawesome icons to icons8 images
- I made final improvements on spacings and style
- Update README.md file

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,712 @@
...
+  grid-column: span 3;
+}
+
+.span4 {
+  grid-column: span 4;
+}
+
+/* .icons {
+  fill: #fff;
+} */
```
Comment (kevcap) [inline css/style.css]: - [x] Remove the commented code 🗑

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+  font-size: 42px;
+  background-color: transparent;
+}
+
+.article-title-span h2 span {
+  font-size: 16px;
+}
+
+.hashtags {
+  /* width: 100%; */
```
Comment (kevcap) [inline css/style.css]: - [x] Remove the commented code 🗑

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+.latest-posts {
+  grid-template-rows: 100vh 100vh 100vh 100vh;
+  grid-template-columns: 1fr 1fr 1fr;
+  background-color: #f6f3f2;
+  row-gap: 45px;
+  padding: 15px;
+}
+
+.lp-item {
+  /* border: 1px solid #000; */
```
Comment (kevcap) [inline css/style.css]: - [x] Remove the commented code 🗑

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey @chubaquelo hello again! As always your project is looking amazing buddy, you're really good with details and design congrats! It breaks my heart not aprove it right away, and I mean it. But we just have to fix just TWO minor issues and you're good to go ! 💪
 
_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, to get more knowledge and have a better project to add at your portfolio.

### Me reading your code: 

 
 
## [REQUIRED CHANGES]
- [ ] Please remove all the **commented piece of codes** in your css. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */  

- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image



## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,712 @@
...
+  transition: 0.3s;
+  margin-left: 5px;
+}
+
+.seriously-red {
+  width: 170px;
+  height: 40px;
+  border: 1px solid #fff;
+  border-radius: 10px;
+  color: #fff;
```
Comment (kevcap) [inline css/style.css]: `color: #fff ` This rule repeats a lot (18x) you can create a class for that! 💪

**Review by voscarmv — APPROVED** [review]
# Impressive work! 🖥️ 👀 ✨ 

- Good job applying the changes required by Kevin.

# Status: Project Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**